benchmarks/analysis_3d.py

```diff
@@ -66,13 +86,55 @@ def run_benchmark():
 
     stacked.model = model
 
-    fit = Fit(stacked)
+    fit = Fit([stacked])
     result = fit.run(optimize_opts={"print_level": 1})
 
+
+def flux_point(stacked):
     e_edges = [0.3, 1, 3, 10] * u.TeV
     fpe = FluxPointsEstimator(datasets=[stacked], e_edges=e_edges, source="gc-source")
     fpe.run()
 
 
+def run_benchmark():
+    times = []
```

Comment:
Make an empty dict here, and then after each function, add an entry.
(never use a Python list or Numpy array of time points)

Reply:
Thanks @cdeil ! This works now. I will adapt the rest of the files and finish this PR